src/m365/spo/commands/site/site-recyclebinitem-restore.spec.ts

```diff
@@ -76,6 +77,26 @@ describe(commands.SITE_RECYCLEBINITEM_RESTORE, () => {
     assert.notStrictEqual(actual, true);
   });
 
+  it('fails validation if no options are specified', async () => {
+    const actual = await command.validate({ options: { siteUrl: 'https://contoso.sharepoint.com' } }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('fails validation if ids and allPrimary options are specified together', async () => {
+    const actual = await command.validate({ options: { siteUrl: 'https://contoso.sharepoint.com', ids: '5fb84a1f-6ab5-4d07-a6aa-31bba6de9526', allPrimary: true } }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('fails validation if ids and allSecondary options are specified together', async () => {
+    const actual = await command.validate({ options: { siteUrl: 'https://contoso.sharepoinrestores all items from the first-stage and second-stage recycle binst.com', ids: '5fb84a1f-6ab5-4d07-a6aa-31bba6de9526', allSecondary: true } }, commandInfo);
```

Comment:
Very creative tenant name 😁I highly doubt that this is a valid tenant name. Anyway, there's a typo in it `binst`.